experimental/msal-react/src/index.ts

```diff
@@ -0,0 +1,22 @@
+// TODO: How do we get the token before executing an API call in pure TypeScript, outside of a React component context?
+// TODO: How do we raise the `error` state into the MsalContext, where changes will allow all subscribed components to update?
+// TODO: How do we represent the current state of the authentication process (Authenticated, InProgress, IsError, Unauthenticated)?
```

Comment:
The `AuthenticationState` enum from react-aad-msal may not work the same way in v2, given multiple account support. We will expose an event API to know when an auth event is in progress, and the isAuthenticated hook can answer the question about whether or not you are authenticated.

Reply:
Auth errors are per request, so I'm not sure we want to include a global singleton error state.

Reply:
Should be fine so long as we don't create scenarios where the `forceLogin` prop triggers an endless loop.